core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -151,7 +156,7 @@ public void setNewDeviceRequest(NewDeviceRequest newDeviceRequest)
 		this.newDeviceRequest = newDeviceRequest;
 	}
 
-	private UserPrivate getUserPrivate()
+	public UserPrivate getUserPrivate()
```

Comment:
What do you use this for? Currently, ``User`` hides the existence of ``UserPrivate``.